test/e2e/cypress/e2e/users.cy.js

```diff
@@ -0,0 +1,341 @@
+import { userFactory } from '@lib/test-utils/factories/users';
+
+const PASSWORD = 'password';
+const USER = userFactory.build({ username: 'e2etest' });
+
+const getUserIdFromPath = () =>
+  cy.location().then(({ pathname }) => {
+    return pathname.split('/')[2];
+  });
+
+const deleteAllUsers = () => {
+  cy.apiLogin().then(({ accessToken }) =>
+    cy
+      .request({
+        url: '/api/v1/users',
+        method: 'GET',
+        auth: { bearer: accessToken },
+        body: {},
+      })
+      .then(({ body: users }) => {
+        users.forEach(({ id }) => {
+          if (id === 1) {
+            return;
+          }
+          cy.request({
```

Comment:
If we want to wait until all the users are really deleted, we could map on users and accumulate cy.request promises and do then or await on Promise.all